protostar-rust/src/main.rs

```diff
@@ -1,13 +1,23 @@
-use std::env::set_var;
+use std::{collections::HashMap, env::set_var};
 
 use anyhow::{anyhow, Result};
+use cairo_lang_runner::ProtostarTestConfig;
 use camino::Utf8PathBuf;
 use clap::Parser;
 use scarb_metadata::{Metadata, MetadataCommand, PackageId};
 
 use cairo_lang_protostar::test_collector::LinkedLibrary;
 use rust_test_runner::pretty_printing;
-use rust_test_runner::{run_test_runner, ProtostarTestConfig};
+use rust_test_runner::run_test_runner;
+
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug)]
+pub struct ProtostarTestConfigForDeserialization {
+    #[serde(default)]
+    #[warn(dead_code)]
+    exit_first: bool,
+}
```

Comment:
Well, this structure was added for deserialization that was not used in the end. I think we can remove this if we don't use it yet. I left it as is so it will be easier in the future to implement using the config.
This PR is not about deserialization/config and that should be done in a separate PR.

Options I see:
- we remove this
- we leave it as is

Reply:
I would remove it if we don't use it yet, it will result in unnecessary clutter

Reply:
Okay, we can remove it. Just let's remember that `ProtostarTestConfig` should be used directly for deserialization once it is needed, without any extra structs.

Reply:
[remove unnecessary config](https://github.com/software-mansion/protostar/pull/2059/commits/3ff2bdfb5fde9ceec5982f58545177381d65a0cb)